### DISCUSSION TITLE
Please add yellow color to accent colors

### DISCUSSION THREAD

**Discussion by MilutinMujovic** [ubuntu/yaru#3659] [Ideas]
Hello, as someone who generally uses yellow in theming, I am sad that there isn't a yellow color in the accent color selection.
Please add it.

Thank you.

**Comment by meetdilip** [comment]
I just worked a bit on the SVG under 

icons > src > fullcolor > accented > places

and it looks like this now for yellow accent. If the team feel like proceeding, I can try and convert them all. 

![image](https://user-images.githubusercontent.com/6965443/193409844-8fd18032-fbbf-4b57-8411-6021cce56a1a.png)

**Reply by Jupi007** [reply]
Thanks @meetdilip, but all our accent stuff is auto generated :)

**Reply by meetdilip** [reply]
Understood :)

**Reply by Muqtxdir** [reply]
id like to point out that yellow is very problematic color in terms of contrast for both dark and light backgrounds. As you may remember in the initial stages of the palette for accent colors also had yellow and was later on discarded for this reason.

**Reply by meetdilip** [reply]
I guess yellow would need a separate file in that case. I did more than changing green to yellow. A few adjustments with gradients and how they are added.

**Reply by Muqtxdir** [reply]
that's what I'm trying to say yellow without any transformations applied on does not have enough contrast for both backgrounds.

also you are just manually editing the svg files, and producing gradient transformations over yellow. so that defeats the whole purpose of automating the accent colors during build

**Reply by LordMilutin** [reply]
It looks nice, just about 10% darker. 
I know this is not pleasing to everyone, but you must understand that not all people see colors the same. For example, I am really having issues with that opaque/brown/sand color as it is hardly readable by my eyes, so this is a great improvement over that. 

Of course, not everyone will use it, but some of us who find it appealing will use it for sure. 

So at least an OPTION to have it would be greatly appreciated! :)
